css/amp.css

```diff
@@ -219,10 +219,10 @@ i-amphtml-sizer {
  * does not allow activation of buttons.
  */
 .i-amphtml-screen-reader {
-  position: fixed !important;
-  /* keep it on viewport */
-  top: 0px !important;
-  left: 0px !important;
+  position: absolute !important;
```

Comment:
let's up the specificity of this class by adding body: `body .i-amphtml-screen-reader ` to make it less likely any code can escape the `!important`s here.